src/Scripts/Python/RekoProgramAPI.cs

```diff
@@ -42,97 +42,106 @@ public RekoProgramAPI(Program program)
+            return ReadData(rdr => rdr.ReadByte(), startAddr, length);
         }
 
-        public short ReadInt16(ulong linearAddress)
+        public short ReadInt16(Address addr)
         {
-            return CreateImageReader(linearAddress).ReadInt16();
+            return CreateImageReader(addr).ReadInt16();
         }
 
-        public IEnumerable<short> ReadInts16(ulong iStartAddr, long length)
+        public IEnumerable<short> ReadInts16(Address startAddr, long length)
         {
-            return ReadData(rdr => rdr.ReadInt16(), iStartAddr, length);
+            return ReadData(rdr => rdr.ReadInt16(), startAddr, length);
         }
 
-        public int ReadInt32(ulong linearAddress)
+        public int ReadInt32(Address addr)
         {
-            return CreateImageReader(linearAddress).ReadInt32();
+            return CreateImageReader(addr).ReadInt32();
         }
 
-        public IEnumerable<int> ReadInts32(ulong iStartAddr, long length)
+        public IEnumerable<int> ReadInts32(Address startAddr, long length)
         {
-            return ReadData(rdr => rdr.ReadInt32(), iStartAddr, length);
+            return ReadData(rdr => rdr.ReadInt32(), startAddr, length);
         }
 
-        public long ReadInt64(ulong linearAddress)
+        public long ReadInt64(Address addr)
         {
-            return CreateImageReader(linearAddress).ReadInt64();
+            return CreateImageReader(addr).ReadInt64();
         }
 
-        public IEnumerable<long> ReadInts64(ulong iStartAddr, long length)
+        public IEnumerable<long> ReadInts64(Address startAddr, long length)
         {
-            return ReadData(rdr => rdr.ReadInt64(), iStartAddr, length);
+            return ReadData(rdr => rdr.ReadInt64(), startAddr, length);
         }
 
-        public string ReadCString(ulong linearAddress)
+        public string ReadCString(Address addr)
         {
-            var rdr = CreateImageReader(linearAddress);
+            var rdr = CreateImageReader(addr);
             var c = rdr.ReadCString(PrimitiveType.Char, program.TextEncoding);
             return c.ToString();
         }
 
-        public void SetUserComment(ulong linearAddress, string comment)
+        public void SetUserComment(Address addr, string comment)
         {
-            var addr = Addr(linearAddress);
             program.User.Annotations[addr] = NormalizeLineEndings(comment);
         }
 
-        public void SetUserGlobal(ulong linearAddress, string decl)
+        public void SetUserGlobal(Address addr, string decl)
         {
-            var addr = Addr(linearAddress);
             var usb = new UserSignatureBuilder(program);
             var global = usb.ParseGlobalDeclaration(decl);
             var name = global?.Name;
             var dataType = global?.DataType;
             if (name is null || dataType is null)
                 throw new ArgumentException(
                     $"Failed to parse global variable declaration: '{decl}'");
```

Comment:
Please add a '.' to the end of exception messages, for consistency.

Reply:
Done in 608181f